README.md

```diff
@@ -9,12 +9,14 @@ To add this email-notification-action to your builds, just add the following in
 ```
    - name: Send email notification
      if: github.ref == 'refs/heads/master' && always()
-     uses: byzgenltd/email-notification-action@v1
+     uses: byzgenltd/email-notification-action@v2
```

Comment:
do github actions not use semantic versioning?

Reply:
the only reason I didn't add changelog is I am not expecting anymore changes in it.